pyan/analyzer.py

```diff
@@ -1,19 +1,16 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
-"""The AST visitor.
+"""The AST visitor."""
 
-Created on Mon Nov 13 03:33:00 2017
-
-Original code by Edmund Horner.
-Python 3 port by Juha Jeronen.
```

Comment:
Is there a reason to remove the attributions here?

Reply:
Good question!

If I recall, I thought that the author information is already collected into `README.md`, so it's redundant to have it in `analyzer.py`, especially since Pyan is no longer a single-module project. This makes the docstring cleaner.

If you think it's better to have the author information also in the individual file, I can put the information back. (But no timestamp - Pyan is much older than Mon Nov 13 2017 :) )

To do this properly, we could include a separate `AUTHORS.md` file listing the contributors and their contributions (moving this information from the current `README.md`), but whichever solution you prefer is fine with me.